RELEASE.md

```diff
@@ -1,6 +1,6 @@
-# RAJAProxies 0.1
```

Comment:
@rhornung67 Is `v1.1.0` ok for the release version?

Reply:
I would make it 2.0.0 since you added HIP to CoMD.